gokart/utils.py

```diff
@@ -7,3 +11,48 @@ def add_config(file_path: str):
     _, ext = os.path.splitext(file_path)
     luigi.configuration.core.parser = ext
     assert luigi.configuration.add_config_path(file_path)
+
+
+T = TypeVar('T')
+if sys.version_info >= (3, 10):
+    from typing import TypeAlias
+
+    FlattableItems: TypeAlias = T | Iterable['FlattableItems[T]'] | dict[str, 'FlattableItems[T]']
```

Comment:
TypeAlias is required in mypy. from Python 3.12, this will be replaced [`type statement`](https://docs.python.org/ja/3/reference/simple_stmts.html#type)